tests/foreman/longrun/test_inc_updates.py

```diff
@@ -207,28 +200,25 @@ def test_positive_noapply_api(
     versions = sorted(module_cv.read().version, key=lambda ver: ver.id)
     cvv = versions[-1].read()
     cvv.promote(data={'environment_ids': dev_lce.id})
-    # Read CV to pick up LCE ID and next_version
-    module_cv = module_cv.read()
 
-    # Get the content view versions and use the recent one. API always
-    # returns the versions in ascending order (last in the list is most recent)
-    cv_versions = module_cv.version
     # Get the applicable errata
     errata_list = get_applicable_errata(custom_repo)
     assert len(errata_list) > 0
 
     # Apply incremental update using the first applicable errata
-    entities.ContentViewVersion().incremental_update(
+    outval = entities.ContentViewVersion().incremental_update(
```

Comment:
We should update any use of `entities` to `target_sat` or `module_target_sat`

Reply:
fyi.. replacement of entities with module_target_sat.api will be taken care by Omkar. So I am not changing non affected part of test module.